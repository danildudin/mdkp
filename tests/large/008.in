50 19 0
86 900 34 519 792 59 297 664 907 237 770 755 835 68 504 26 704 277 12 505 306 36 188 168 370 466 543 300 182 962 153 86 257 586 499 348 71 200 148 483 867 154 833 824 58 229 593 540 539 303 
726 309 370 385 796 521 623 996 227 821 661 514 440 589 746 625 454 27 295 506 538 750 449 889 573 618 602 425 479 454 156 391 51 260 529 284 288 661 310 806 866 5 632 747 587 550 783 641 476 528 
387 851 183 616 675 981 558 683 897 605 836 626 126 644 581 539 801 875 111 369 481 77 256 545 168 664 896 303 674 593 368 230 855 110 833 999 857 63 485 643 616 925 660 512 663 639 430 799 968 298 
775 16 863 684 296 861 797 788 390 150 369 872 661 494 922 963 617 196 909 226 10 861 946 353 4 596 845 983 504 623 177 316 118 37 117 525 65 584 524 535 925 201 528 193 191 964 414 327 215 812 
130 160 566 860 847 402 833 585 197 112 157 401 880 432 278 883 152 385 673 420 737 538 161 63 29 495 619 864 553 394 89 22 254 985 409 668 334 292 509 371 128 424 163 322 543 423 627 178 474 615 
873 892 426 926 528 427 298 358 758 909 635 79 621 392 840 439 198 511 402 453 246 900 144 362 894 884 632 420 243 460 366 340 627 253 391 37 517 720 177 575 845 416 90 123 347 824 677 59 75 570 
745 487 997 784 669 525 635 297 609 33 225 745 203 19 157 32 284 56 654 593 674 479 681 455 591 640 993 752 901 728 13 151 744 366 36 633 685 776 993 276 923 897 586 210 788 520 672 183 362 843 
799 679 644 437 406 995 750 932 500 691 894 151 556 267 532 722 191 803 780 111 528 509 569 853 488 748 820 506 640 250 255 372 914 901 73 258 742 405 815 921 425 256 136 855 652 283 937 122 271 127 
883 744 134 441 942 101 405 269 8 209 629 908 238 945 322 250 559 650 674 431 173 641 440 509 237 330 261 935 276 3 676 563 862 838 12 615 593 374 428 638 868 75 839 184 194 698 661 852 11 470 
253 611 436 928 144 351 117 895 290 912 358 415 830 330 124 249 710 539 709 121 129 368 885 277 466 136 684 35 561 574 130 518 180 778 925 123 504 678 705 403 144 859 165 836 883 227 900 868 198 669 
756 201 180 238 981 893 443 137 915 907 336 339 432 824 630 814 798 880 900 816 8 237 261 388 563 863 642 691 902 451 136 556 542 595 790 531 322 339 99 564 297 881 942 838 522 738 431 637 567 489 
655 603 935 644 491 451 61 123 275 723 344 475 114 846 642 990 803 878 583 251 590 746 875 676 585 164 213 609 660 345 110 667 961 409 842 507 504 344 371 262 934 672 26 972 626 469 72 304 683 280 
456 147 200 23 787 114 751 141 933 951 259 982 103 985 949 946 931 164 984 345 974 985 528 769 947 980 538 905 795 732 310 720 966 530 596 451 687 715 555 512 594 34 906 816 602 877 698 118 142 875 
794 157 947 835 583 451 52 326 687 672 577 107 718 580 784 567 639 457 103 374 565 868 435 942 662 437 834 785 730 333 101 578 731 8 518 393 306 255 95 774 977 636 910 589 932 753 583 846 816 822 
106 129 650 35 703 360 935 302 866 506 405 311 791 573 422 519 275 942 893 458 796 602 128 594 985 690 862 494 76 366 658 220 195 778 778 518 181 50 198 202 176 22 239 781 465 152 361 769 828 799 
33 933 537 532 499 78 6 596 188 985 284 850 337 236 638 644 361 638 177 852 361 808 870 812 406 286 212 40 619 961 610 12 811 658 840 400 436 464 10 44 429 987 942 385 755 813 287 792 126 213 
842 463 397 867 250 762 882 695 83 218 309 936 753 698 598 429 815 174 10 171 648 883 502 355 146 459 398 300 56 510 971 194 871 545 801 518 142 123 19 529 275 205 280 336 550 61 768 238 750 540 
155 154 789 975 153 534 566 392 492 667 513 361 538 168 406 823 78 660 630 887 352 367 208 94 929 609 482 183 575 445 196 48 514 590 954 493 275 689 562 663 921 481 813 724 997 606 400 177 552 959 
999 423 502 815 449 319 616 717 672 84 98 726 140 395 327 430 337 976 204 46 658 683 297 511 356 713 842 295 720 403 604 619 59 855 90 130 205 188 159 121 428 457 78 1000 727 220 408 654 28 603 
956 975 766 758 782 516 79 5 825 11 504 349 437 542 123 171 881 114 757 525 315 858 490 472 535 168 569 894 812 466 658 524 851 116 875 59 89 864 69 689 776 911 736 268 49 829 870 286 893 491 
84272 60300 57189 65044 44439 21402 21743 12134 52072 95028 36447 24678 72556 60690 85817 82478 41220 79246 11062 
