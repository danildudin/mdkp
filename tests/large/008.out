15502
26
1 4 6 7 8 9 10 11 12 14 16 17 19 20 25 26 29 33 35 39 40 42 43 44 47 48 
